boskos/janitor/deployment.yaml

```diff
@@ -15,10 +15,10 @@ spec:
       terminationGracePeriodSeconds: 300
       containers:
       - name: boskos-janitor
-        image: gcr.io/k8s-testimages/janitor:v20170830-c0d24857
+        image: gcr.io/k8s-testimages/janitor:v20170921-b5302345
         args:
         - --service-account=/etc/service-account/service-account.json
-        - --resource-type=gce-project,gke-project
+        - --resource-type=gce-project,gke-project,gci-qa-project,gpu-project
```

Comment:
(the addition with `gci-qa-project` is actually deployed but I guess I have never committed it :-( )